### ISSUE TITLE
Two animations at the same time?

### ISSUE THREAD

**Issue by falafflepotatoe** [anijs/anijs#44]
Sorry quick questions for you

How do i run two anims at the same time, is that possible with the anijs-data language?

How do i run one anim after another without using onanimationend? (because the elem has a click and load animation -- so i cant really use onanimationend)

How do I hide an elem in the after: section? (or add class)

**Comment by falafflepotatoe** [comment]
i figured out on the last question - after: $removeClass x works if i have to:target in the line. but in chrome before: $removeClass x doesnt work it says

Uncaught TypeError: Cannot call method 'replace' of undefined anijs-min.js:1

probably because its doing getElementbyId('').attr('class').replace()

**Comment by darielnoel** [comment]
Hi, thanks for feedback :)

1- Yes, you can run two anims at the same time

```html
<p data-anijs="if:click, do: animation1 animation2 animation3"></p>
```

2 -  Sorry , I don't understand your explanation :(. Could you please give more details?

3- Yes, you are rigth, you could use **$removeClass** x or **$addClass** for adding or removing classes in the after: section.

3.1 - He [probado](http://codepen.io/darielnoel/pen/NPwvyy) to do **before: $removeClass x** in chrome and firefox and works fine. Maybe I miss me something to reproduce the error correctly. You could help by making a small example in [codepen](http://codepen.io/)? 

Thank you very much for the feedback. if you need anything else please write.

Best Regards, Dariel

**Comment by falafflepotatoe** [comment]
sure - about #2.
i guess this is just a feature request - i could probably do this with creating a custom animation. but it would be great to do animation chains:

(on 1 element)
if:load, do: bounce, then: pulse
if:click, do: rubberBand, then: delay 1s, then: do bounce infinite

i dont think its working now because i do
if load do bounce, on animation end do pulse
if click do rubberband, on animation end do: fadeOut

on the second animation end -- i think -- its gonna do pulse or nothing - i dont think its gonna do fade out because it cant tell (?) which animation has just ended.

re: #3 
http://codepen.io/anon/pen/ZYaaRa

not sure if its because im watching load -- or because i have two events watched and 2 strings to parse -- or i put the after/before cmds in the wrong place.  probably my fault ^_^

re: #1
thanks!

**Comment by falafflepotatoe** [comment]
after tinkering with it some more, it looks like if: load and before:removeClass dont work together. i'm guessing load isnt a real event to use =P

sorry to waste ur time

what do you do to animate on page load usually?

**Comment by darielnoel** [comment]
1-  I understand, could be a great feature :). By the moment, you can solve it creating a custom animation as you say.

2- There are some details in your sentence

2.1- You need to specify the **load** event owner which is the **window** object.

```html
<header data-anijs="if: load, on: window, before: $removeClass displayNone, do: flipInY animated, after: $addClass test, to: target"></header>
```

2.2 You don't might to put **to:target**. Because the target is the element who fire the event. In this case **window**.

```html
<header data-anijs="if: load, on: window, before: $removeClass displayNone, do: flipInY animated, after: $addClass test"></header>
```
[Here](http://codepen.io/darielnoel/pen/emeyVx) the example running.

Also I found a small bug in the **dom-helper** code. Which don't allow to execute the **before** functions properly. I fixed it, you can find the details in this [commit](https://github.com/anijs/anijs/commit/c941100586f03f448b91d435edeb7e845b13c46c).

Don't forget to update AniJS to [0.9.1 version](https://github.com/anijs/anijs/releases/tag/v0.9.1).

Thanks again, for your kind collaboration :)

**Comment by falafflepotatoe** [comment]
cool thanks

cheers :)

**Comment by darielnoel** [comment]
Hi, we launched [AniJS Studio](https://www.kickstarter.com/projects/888703031/anijs-studio) on KickStarter, that allow to designers and programmers add animations and interactions for Web pages in an easy and intuitive way. Please, take a look at the [project](https://www.kickstarter.com/projects/888703031/anijs-studio) and if you find it interesting or simply want to help us, you can make a small donation. I would be grateful, Greetings Dariel

**Comment by iocouto** [comment]
@darielnoel perhaps you could create a new issue for a "[Feature Request] Chained Animations" - which is the only thing left unanswered in this thread. You could then close this issue.
